packages/better-auth/src/plugins/passkey/client.ts

```diff
@@ -153,6 +153,32 @@ export const getPasskeyActions = (
 		}
 	};
 
+	const updatePasskey = async (
+		opts: {
+			fetchOptions?: BetterFetchOption;
+			id: string;
+			name: string;
+		},
+		fetchOpts?: BetterFetchOption,
```

Comment:
I don't understand why fetchOpts is passed twice.

Reply:
Honestly, neither do I :sweat_smile: . I was just following along the structure of the previous methods in the code. And they were passing it twice just like this. Maybe that is something that should be looked at on all the current methods.